test/ng/directive/ngModelSpec.js

```diff
@@ -19,7 +19,6 @@ describe('ngModel', function() {
       };
 
       element = jqLite('<form><input></form>');
-      element.data('$formController', parentFormCtrl);
```

Comment:
Since the parentFormCtrl is now extracted via regular require, this mechanism doesn't work anymore. This might break people's tests.